pkg/node/address.go

```diff
@@ -806,8 +807,29 @@ func copyStringToNetIPMap(in map[string]net.IP) map[string]net.IP {
 	return out
 }
 
+// WithTestLocalNodeStore sets the 'localNode' to a temporary instance and
+// runs the given test. Afterwards the 'localNode' is restored to nil.
+// This is a temporary workaround for tests until the LocalNodeStoreCell can be
+// used.
+func WithTestLocalNodeStore(runTest func()) {
```

Comment:
I'm trying this approach out as having a default usable instance is scary since any calls to `node.Set*` before `LocalNodeStore` is started will be thrown out. It's better to have silly wrappers in tests than have bugs in the production code.